python/lsst/ts/ATHexapod/mock_server.py

```diff
@@ -278,7 +278,7 @@ def __init__(self, log=None):
 
     async def check_offset(self, x, y, z, u, v, w):
         """Check that the hexapod can move."""
-        return (f"X=1 Y=1 Z=1 U=1 V=1 W=1\n")
+        return (f"X=1\n Y=1\n Z=1\n U=1\n V=1\n W=1\n")
```

Comment:
Does the real hexapod actually output this, or the old  string?

Reply:
This function is meant to "replicate" the virtual move check that the hexapod does in order to make sure that the move works in real life. However, I don't actually validate the movement in the mock server, so this just assumes everything's hunky dory.

Reply:
I don't understand your answer, and I wonder if you misunderstood my question. What I am asking is: does the real hexapod output data like the old string you had here (without newlines) or the new string (with newlines)?